lib/go-qmstr/cli/create.go

```diff
@@ -16,27 +16,27 @@ var cmdFlags *pflag.FlagSet
 
 var createCmd = &cobra.Command{
 	Use:   "create",
-	Short: "create a new node",
+	Short: "Create a new node",
 	Long:  "create a new node described by a node identifier",
```

Comment:
Why does short description start capitalized and long does not?

Reply:
For the moment, consistency. All the other descriptions I found are the same way.